POEApi.Transport/HttpTransport.cs

```diff
@@ -36,6 +36,8 @@ protected enum HttpMethod { GET, POST }
         private const string UpdateShopURL = @"https://www.pathofexile.com/forum/edit-thread/{0}";
         private const string BumpShopURL = @"https://www.pathofexile.com/forum/post-reply/{0}";
 
+        private const int _maximumWindowLimit = 42;

```

Comment:
Can this be brought in as an application setting?

Reply:
I would rather not, for two reasons.
1) This value will likely very rarely -- if ever -- change, so the user will be unlikely to modify the setting.  He can also easily make Procurement work very poorly by increasing the value.
2) This class can't reference the `Settings` directly, since that would introduce a circular dependency between the projects.  Thus -- barring more substantial refactoring I'm not interested in doing now -- it would then have to be set after the static constructor finishes, potentially after it started handling requests.